ocw_import/api.py

```diff
@@ -145,13 +171,73 @@ def convert_data_to_content(
         if not uid:
             log.error("No UUID (text ID): %s", filepath)
         else:
-
             content, _ = WebsiteContent.objects.update_or_create(
                 website=website, text_id=str(uuid.UUID(uid)), defaults=base_defaults
             )
             return content
 
 
+# pylint:disable=unused-argument
+def convert_json_to_resource(filepath="", data="", website=None, **kwargs):
+    """
+    Convert json data into a WebsiteContent object
+
+    Args:
+        filepath(str): The path of the file (from S3 or git)
+        data(str): The file data to be converted
+        website: The website to which the content belongs
+    """
+    return save_resource(
+        filepath=filepath, extension="json", data=data, website=website
+    )
+
+
+# pylint:disable=unused-argument
+def convert_toml_to_resource(filepath="", data="", website=None, **kwargs):
+    """
+    Convert toml data into a WebsiteContent object
+
+    Args:
+        filepath(str): The path of the file (from S3 or git)
+        data(str): The file data to be converted
+        website: The website to which the content belongs
+    """
+    return save_resource(
+        filepath=filepath,
+        extension="toml",
+        data=json.dumps(toml.loads(data)),
+        website=website,
+    )
+
+
+def save_resource(filepath, extension="", data="", website=None):
+    """
+    Save a WebsiteContent object of type CONTENT_TYPE_RESOURCE
+
+    Args:
+        filepath(str): The path of the file (from S3 or git)
+        extension(str): The file extension of the file to be saved
+        data(str): The file data to be converted
+        website: The website to which the content belongs
+    """
+    uid = uuid.uuid4()
+    dirpath, filename = get_dirpath_and_filename(filepath, expect_file_extension=True)
+    base_defaults = {
+        "metadata": data,
```

Comment:
The metadata is being saved as a string, but it should be saved as actual JSON.

<img width="476" alt="Screen Shot 2021-05-25 at 9 23 22 AM" src="https://user-images.githubusercontent.com/187676/119505712-3b448b00-bd3b-11eb-8c71-68b05ea39842.png">